src/app/courses/add-course/add-course.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-add-course',
+  templateUrl: './add-course.component.html',
+  styleUrls: ['./add-course.component.scss']
+})
+export class AddCourseComponent implements OnInit {
+
+  title: string;
+  description: string;
+  duration: string;
+  date: string;
```

Comment:
what about bindings?

Reply:
What do You mean exactly?
Unless ngModel is a binding, is not?
![image](https://user-images.githubusercontent.com/11742962/101278981-9d2e2a00-37c7-11eb-93fb-23c389626607.png)